packages/Tabs/src/components/Tab/Tab.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import RawButton from "@paprika/raw-button";
+import TabsContext from "../../TabsContext";
+import tabStyles from "./Tab.styles";
+
+const propTypes = {
+  className: PropTypes.string,
+  label: PropTypes.node,
+  isSelected: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  linkUrl: PropTypes.string,
+  onClick: PropTypes.func,
+  onKeyDown: PropTypes.func,
+  onKeyDownArrows: PropTypes.func,
+};
+
+const defaultProps = {
+  className: null,
+  isSelected: false,
+  isDisabled: false,
+  label: null,
+  linkUrl: null,
+  onClick: () => {},
+  onKeyDownArrows: () => {},
+  onKeyDown: () => {},
+};
+
+const Tab = props => {
+  const context = React.useContext(TabsContext);
+
+  const {
+    className,
+    isDisabled,
+    isSelected,
+    label,
+    linkUrl,
+    onClick,
+    onKeyDown,
+    onKeyDownArrows,
+    ...moreProps
+  } = props;
+
+  const _isDisabled = context.isDisabled || props.isDisabled;
+
+  const cn = classNames("tab", { "tab--is-active": isSelected }, { "tab--is-disabled": _isDisabled }, className);
+
+  const handleKeyDown = event => {
+    const leftArrowKey = 37;
+    const rightArrowKey = 39;
+
+    if (event.which === leftArrowKey) {
+      onKeyDownArrows(context.activeIndex - 1);
+    } else if (event.which === rightArrowKey) {
+      onKeyDownArrows(context.activeIndex + 1);
+    }
+
+    onKeyDown(event);
+  };
+
+  const handleClick = _isDisabled ? () => {} : onClick;
+  const tabIndex = _isDisabled ? -1 : 0;
+
+  if (linkUrl) {
+    return (
+      <a
+        css={tabStyles}
+        className={classNames("tab-link", cn)}
+        href={linkUrl}
+        onKeyDown={handleKeyDown}
+        role="tab"
+        tabIndex={tabIndex}
+        {...moreProps}
+      >
+        {label}
+      </a>
+    );
+  }
+
+  return (
+    <RawButton
+      aria-selected={isSelected}
+      css={tabStyles}
+      className={cn}
```

Comment:
🤔 wouldn't more sense to passdown isSelected and _isDisabled to the `css={tabSyles}`? instead of using a className?  to dictate how they should look like?